doc/architecture/20170816-audit-plan-to-model-audit-data.md

```diff
@@ -0,0 +1,29 @@
+
+
+## Term definitions
+
+A **theme** is a high level grouping criteria for all the content that GOV.Uk is currently hosting.
+
+An **inventory plan** is a set of rules that define the content that belongs to a theme. 
```

Comment:
As of now, content items do not move across themes. The inventory plan is just a set of rules.